flyte/pack.go

```diff
@@ -102,6 +102,7 @@ func (p pack) Start() {
 		p.Start()
 		return
 	}
+	log.Info().Msgf("%s pack is registered and available to take action", p.Name)
```

Comment:
```suggestion
	log.Info().Msgf("%s pack has registered with flyte api", p.Name)
```

This bit is irrelevant since pack can do more things than just take actions.
`and available to take action`